catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -122,11 +122,29 @@ public UpdateRequest process(UpdateRequest input)
         if ((isValidatorEnforced(validatorName) && validationErrorsExist && enforceErrors)
             || isValidatorEnforced(validatorName) && validationWarningsExist && enforceWarnings) {
           INGEST_LOGGER.debug(
-              "The metacard with id={} is being removed from the operation because it failed the enforced validator [{}].",
+              "The metacard with title={} and id={} is being removed from the operation because it failed the enforced validator [{}].",
+              metacard.getTitle(),
               metacard.getId(),
-              validatorName);
+              validatorName,
+              e);
           return null;
         } else {
+          if (validationErrorsExist) {
+            INGEST_LOGGER.debug(
+                "The metacard with title={} and id={} had an unenforced validation error [{}].",
+                metacard.getTitle(),
+                metacard.getId(),
+                validatorName,
+                e);
```

Comment:
✏️ These stack traces really busy up the logs when ingesting many metacards. Maybe add another `{}` and pass in `e.getMessage()` *instead of* passing in the whole exception.

Reply:
fixed